src/tools/analytics.py

```diff
@@ -29,6 +30,7 @@ def getParameterInfo(self):
             aoi.buffer_parameter(),
             schedule.start_date_parameter(),
             schedule.end_date_parameter(),
+            customer_notes.customer_notes_parameter(),
```

Comment:
What do you think about moving this to the last parameter in the form rather than in the middle? 

Either option isn't bad UX, so if the form looks visually better having the field up here, that's fine by me, but I always think of additional notes to be the last thing a user would want to fill out before hitting submit, so maybe it makes sense to move it down? I'm not sure how it would look with the additional CD options though, so feel free to play around and let me know what you think

Reply:
My main thought when deciding to put it in the middle was the change detection selection box. I wasn't sure if it would be better to have the additional notes be last and then moved further down when the CD selection box appeared or if it would be less disruptive to have it above the analytics selection box and not influenced by the visibility of the CD selection box.

I don't have a strong preference either way.

Reply:
Yeah, that's my hesitation too. it might feel a little awkward to shift it down when the CD selection box appears. Let's go ahead and leave it as is for now